setup.py

```diff
@@ -127,9 +126,9 @@
 CI_REQUIRED = (
     [
         "build",
-        "cryptography==35.0",
+        "cryptography>=35.0,<36",
         "flake8",
-        "black==19.10b0",
+        "black>=22.6.0,<23",
```

Comment:
Is this what caused all the formatting changes?

Reply:
Yeah. I added this in the "Additional notes" section in the PR description. Essentially, the older version of black is not compatible with the newer version of click.